test/integration/1_services/CaverService.test.js

```diff
@@ -92,38 +92,67 @@ describe('CaverService', () => {
       should(caver).equal(undefined);
     });
     it('should return a partial view of the caver when providing an user token', async () => {
-      const caver = await CaverService.getCaver(1, userReq);
-      should(caver.name).equal('Adrien');
-      should(caver.nickname).equal('Admin1');
-      should(caver.surname).equal('Admo');
+      const caver = await CaverService.getCaver(6, userReq);
+      should(caver.id).equal(6);
+      should(caver.name).equal('Axel');
+      should(caver.nickname).equal('Caver1');
+      should(caver.surname).equal('Cavo');
       should(caver.documents.length).equal(3);
       should(caver.documents).containDeep([{ id: 1 }, { id: 2 }, { id: 4 }]);
+      should(caver.groups.length).equal(1);
+      should(caver.groups).containDeep([{ id: 1 }]);
+      should(caver.exploredEntrances.length).equal(1);
+      should(caver.exploredEntrances).containDeep([{ id: 4 }]);
+      caver.exploredEntrances.forEach((entrance) => {
+        should(entrance.isPublic).equal(true);
+      });
+      should(caver.exploredEntrances);
+      should(caver.language).equal('fra');
       should.not.exist(caver.password);
       should.not.exist(caver.activationCode);
     });
+
     it('should return a partial view of the caver when not providing a token', async () => {
-      const caver = await CaverService.getCaver(1, {});
-      should(caver.name).equal('Adrien');
-      should(caver.nickname).equal('Admin1');
-      should(caver.surname).equal('Admo');
+      const caver = await CaverService.getCaver(6, {});
+      should(caver.id).equal(6);
+      should(caver.name).equal('Axel');
+      should(caver.nickname).equal('Caver1');
+      should(caver.surname).equal('Cavo');
       should(caver.documents.length).equal(3);
       should(caver.documents).containDeep([{ id: 1 }, { id: 2 }, { id: 4 }]);
+      should(caver.groups.length).equal(1);
+      should(caver.groups).containDeep([{ id: 1 }]);
+      should(caver.exploredEntrances.length).equal(1);
+      should(caver.exploredEntrances).containDeep([{ id: 4 }]);
+      caver.exploredEntrances.forEach((entrance) => {
+        should(entrance.isPublic).equal(true);
+      });
+      should(caver.language).equal('fra');
       should.not.exist(caver.password);
       should.not.exist(caver.activationCode);
     });
     it('should return a complete view of the caver', async () => {
```

Comment:
it('should return a complete view of the caver **when providing an admin token',** async () => {